config/jobs/kubernetes/sig-release/cip/cloudbuilder.yaml

```diff
@@ -0,0 +1,20 @@
+promoter:
+  setup.sh: |
+    #!/bin/sh
+    set -o errexit
+    set -o nounset
+    set -o pipefail
+    set -o xtrace
+
+    shopt -s dotglob
+
+    git \
```

Comment:
When you write the e2e prow job, it will have a line in the yaml config that says `decorate: true` which will do the cloning of the promoter repo automatically before the job starts. So this cloning is actually not necessary.

Reply:
Yeah most of this is unnecessary, see https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md#what-the-test-container-can-expect